packages/common/storybook-static/iframe.html

```diff
@@ -0,0 +1,69 @@
+<!doctype html><html><head><meta charset="utf-8"><title>Storybook</title><meta name="viewport" content="width=device-width,initial-scale=1"><base target="_parent"><style>:not(.sb-show-main) > .sb-main,
```

Comment:
you should exclude storybook-static from git.

Reply:
🤦🏻‍♀️

True! Thanks!